cluster-lifecycle/update-managedcluster-status-to-include-clusterinfo.md

```diff
@@ -0,0 +1,231 @@
+# Update ManagedCluster status to include ClusterInfo
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Constraints](#constraints)
+- [Design Details](#design-details)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+The proposed work would integrate the ClusterInfoStatus into ManagedCluster:
+
+https://github.com/open-cluster-management/multicloud-operators-foundation/blob/master/pkg/apis/internal.open-cluster-management.io/v1beta1/clusterinfo_types.go#L22-L62
+
+## Motivation
+
+Today, several label properties are updated on ManagedCluster objects that can be modified by users, creating invalid data for matching behavior.
```

Comment:
i think we need to specify what labels are. Also we need to say we want a extensible and neutral way that can work with  different implementations

Reply:
Done.